### ISSUE TITLE
Hub homepage

### ISSUE THREAD

**Issue by mtorres253** [18F/hub#525]
As an 18F employee, so that I have a canonical place to go to _start_ my search for information, or to get guidance on where to find important things, there should be a homepage that acts as an entry point to this information

Acceptance Criteria:
We want to make sure that the new layout/design of the Hub appears at hub.18f.gov, and is hooked up to the new search implementation.

1. Search area
2. Cross-browser test and make sure it works in mobile (latest version of Chrome, IE 9 and above, latest Firefox, latest Safari)

**Comment by mtorres253** [comment]
![image](https://cloud.githubusercontent.com/assets/100149/12599456/efff3552-c446-11e5-96ce-7fddea1a766e.png)

**Comment by andrewmaier** [comment]
That list is a good start. This design also calls for 
1. A list of all tags (is that feasible if we don't store any content on the Hub?)
2. A list of all guides
3. A list of the most popular pages on the Hub (or search terms)

I think I'll need to design a search results page similar to [what Jessie shared earlier today](https://18f.slack.com/files/jessie/F0KECPUHF/screen_shot_2016-01-26_at_10.24.54_am.png)

**Comment by mtorres253** [comment]
@andrewmaier I have a separate story for search results page. This is just the homepage. We should also talk about what to do with existing Hub content that is not in the Handbook

**Comment by mtorres253** [comment]
Waiting on #523 and #535

**Comment by andrewmaier** [comment]
Still waiting on @jbarnicle (re #523) and @catherinedevlin re (#535). I'll update the Hub homepage with a footer based on our design for the Handbook :)

**Comment by mtorres253** [comment]
We still need to discuss whether we want auth for this initial release

**Comment by mtorres253** [comment]
this seems like it's done. Can we move to in review or done? @andrewmaier

**Comment by andrewmaier** [comment]
Done from a design standpoint. I need to add guides to the homepage.

**Comment by mtorres253** [comment]
That's another story https://github.com/18F/hub-v2/issues/4
